clio-client/src/main/scala/org/broadinstitute/clio/client/webclient/ClioWebClient.scala

```diff
@@ -249,10 +249,11 @@ class ClioWebClient(
     */
   def simpleQuery[I](clioIndex: ClioWebClient.QueryAux[I])(
     input: I,
-    includeDeleted: Boolean
+    includeDeleted: Boolean,
+    includeAll: Boolean
   ): Source[Json, NotUsed] = {
     import clioIndex.implicits._
-    if (includeDeleted) {
+    if (includeDeleted || includeAll) {
```

Comment:
So is the idea here to keep includeDeleted to not break things that use it, but really it includes all records, so add an option that is semantically correct?

Reply:
`includeDeleted` always included all results, not just `Normal` and `Deleted` ones. I'd like to replace it with `includeAll` because its much more descriptive of what actually happens, but I'm leaving `includeDeleted` in for compatibility reasons.